### PR TITLE
Wiggle sort

### PR DESCRIPTION
**Pull request by BriseBalloches**
### **Describe your change:**

Improved wiggle_sort.py, mainly with type hinting and doctests.

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [X] Documentation change?

### **Checklist:**
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [X] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: sorts/wiggle_sort.py [hunk 1]
```diff
@@ -9,18 +9,28 @@
 """
 
 
-def wiggle_sort(nums):
-    """Perform Wiggle Sort."""
-    for i in range(len(nums)):
+def wiggle_sort(nums: list):
```
Comment (mateuszz0000) [inline sorts/wiggle_sort.py:12]: Please add return hint. Please also provide doctest with float numbers.

**Review by mateuszz0000 — CHANGES_REQUESTED** [review]

File: sorts/wiggle_sort.py (hunk 1 above)
Comment (BriseBalloches) [reply sorts/wiggle_sort.py]: Done.

**Comment by cclauss** [comment]
@mateuszz0000 If this is ready to merge then please approve and then click Squash & merge.

**Review by mateuszz0000 — APPROVED** [review]

**Comment by cclauss** [comment]
@mateuszz0000 Thanks.  Are there any other open PRs that you think should be landed?

**Pull request merged**